apps/createsamples/utility.cpp

```diff
@@ -995,12 +995,13 @@ void icvGetNextFromBackgroundData( CvBackgroundData* data,
         {
             round = data->round;
 
+            data->last = rand() % data->count;
+            data->last %= data->count;
```

Comment:
IMHO the second line do nothing since the result of the first line is already a remainder. I think it would be better to remove it.

Reply:
@terfendail  Sure, I will remove the second line.